docs/api/config/js_kanban_editorshape_config.md

```diff
@@ -168,7 +168,51 @@ To set the control for assigning one user, you need to use the ***"select"*** or
 
 #### - Parameters for a "files" type
 
-- `uploadURL` - (optional) an URL of the editor uploader
+- `uploadURL` - (optional) an URL of the editor uploader. See the detail below
```

Comment:
sorry, but I can't help it: a URL. the first sound in URL is j, it's a consonant.